### PR TITLE
Render delete for oracle

### PR DESCRIPTION
**Pull request by bburdiliak**
implements #156 

I had to specially deal with boolean values, as Oracle does support them only via smallint (or char).

### CODE REVIEW

**Comment by jczuchnowski** [comment]
> I had to specially deal with boolean values, as Oracle does support them only via smallint (or char).
Yeah, same problem with SQL Server.

This looks good. Thank you!

**Review by jczuchnowski — APPROVED** [review]

File: oracle/src/test/scala/zio/sql/oracle/OracleModuleSpec.scala [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package zio.sql.oracle
+
+import zio.test.Assertion._
+import zio.test.TestAspect._
+import zio.test._
+
+import scala.language.postfixOps
+
+object OracleModuleSpec extends OracleRunnableSpec with ShopSchema {
```
Comment (baldram) [inline oracle/src/test/scala/zio/sql/oracle/OracleModuleSpec.scala:9]: The tested module seems to be called `OracleSqlModule` so the test would be `OracleSqlModuleSpec`.

Comment (baldram) [reply oracle/src/test/scala/zio/sql/oracle/OracleModuleSpec.scala:9]: It's merged so I will rename it within my change.

Comment (jczuchnowski) [reply oracle/src/test/scala/zio/sql/oracle/OracleModuleSpec.scala:9]: :+1:

**Pull request merged**